app/views/reviews/show.html.erb

```diff
@@ -189,24 +189,61 @@
 
 
     <script type="text/javascript">
+      //sets all of the flag colors on first/following page loads and updates
```

Comment:
typically for bugs, if there isn't a test that caught the bug, we should write one.  I don't think you need to do it for the resize of columns (not sure how you'd test that), but this functionality seems testable.